### DISCUSSION TITLE
Error when doing "podman machine init"

### DISCUSSION THREAD

**Discussion by KiwiKinon** [containers/podman#16675] [Q&A] [answered]
Hi,

I'm trying to use Podman but I have an error. When I initialize a machine, I get an error : 
     Error: cannot overwrite connection
Then the machine is created anyway, but when I try to do anything it doesn't work.
When I do "podman pull redhat/ubi8-micro" I get : 
     Error: failed to connect: dial tcp [::1]:49161: connectex: No connection could be made because the target machine actively refused it.

I am being a corporate proxy and I have to make it work with it.

If anyone has an idea it would be great thanks !

**Comment by mheon** [comment]
@ashley-cui PTAL

**Comment by KiwiKinon** [comment, marked as answer]
I've uninstalled Docker from my PC and started again, and it worked
